exampleCourse/questions/demo/workspace/rstudio/workspace/starter_file.md

```diff
@@ -0,0 +1,6 @@
+<!-- prettier-ignore -->
```

Comment:
Why not let Prettier format this file?

Reply:
All right. At first it failed for me, before I edited it, and I thought maybe course staff would fork it and get thrown by that too.